engine/src/dskw32.cpp

```diff
@@ -1435,6 +1532,13 @@ struct MCStdioFileHandle: public MCSystemFileHandle
 	
 	virtual int64_t Tell(void)
 	{
+		/* First flush any write buffer, if this fails, then the operation
+		 * fails. */
+		if (!FlushWriteBuffer())
+		{
+			return false;
```

Comment:
returning a boolean as an int64_t?

Reply:
Copy-paste again :) This is a similar issue to that with Close - currently Close is called under the assumption it cannot fail... But any method which has to flush the buffer to return its result *can fail*. Although, I guess in this case (Tell) it could fetch the position and add the length of the buffer.